eng/linux/Generic/usr/lib/systemd/user/opentabletdriver.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=OpenTabletDriver Daemon
+After=default.target
+
+[Service]
+ExecStart=otd-daemon
+Restart=always
+RestartSec=3
+
+[Install]
+WantedBy=default.target
```

This won't work because we can't ensure that environment variables are ready yet. I suggest the following instead:

```suggestion
[Unit]
Description=OpenTabletDriver Daemon
PartOf=graphical-session.target

[Service]
ExecStart=otd-daemon
Restart=always
RestartSec=3

[Install]
WantedBy=graphical-session.target
```

See "_2) Set up a dependency chain so that your DE starts a systemd target (advanced)_" in https://github.com/OpenTabletDriver/OpenTabletDriver.Web/issues/43#issue-1105884672 for more details.

The alternative for starting should be starting it on DE start, ie. literally `otd-daemon` in the DE/WM's autostart.